src/compiler/checker.ts

```diff
@@ -21078,24 +21231,49 @@ namespace ts {
             return tryGetTypeAtPosition(signature, pos) || anyType;
         }
 
-        function tryGetTypeAtPosition(signature: Signature, pos: number): Type | undefined {
+        function tryGetAtXPositionWorker<T>(

```

Comment:
I don't like this new helper. Just keep the old function the way it was and introduce new functions as appropriate.

Reply:
it's gone now~